api/views.py

```diff
@@ -1,6 +1,65 @@
 from django.http import HttpResponse
+import PyPDF2
+from . import env
+from . import requests
+from PyPDF2 import PdfFileReader 
+from PyPDF2 import PdfFileMerger
+from django.shortcuts import render
+from django.conf import settings
+import os
 
 def consume_file(request):
     if request.method == 'POST':
         print(request.FILES['file'])
-        return HttpResponse(request.FILES['file'].name)
\ No newline at end of file
+        return HttpResponse(request.FILES['file'].name)
+    
+#error handling
+
+def uploadFile(fileFullPath):
```

Comment:
You should do error handling in the method consumeFile (POST request).
You get a file and check the extensions. If extension is not .pdf then return HTTP status code 400.